main.py

```diff
@@ -132,7 +132,7 @@ def repeat_function(cargo_loading_time: float):
         if take_screenshots:
             img = pyautogui.screenshot()
             img.save(
-                f"eve_screenshot_{datetime.now().strftime("%d-%m-%Y-%H-%M-%S")}.png"
+                f'eve_screenshot_{datetime.now().strftime("%d-%m-%Y-%H-%M-%S")}.png'

```

Comment:
This will probably be reformatted by Black. Are you sure its necessary @x-actly ?

Reply:
I was only able to start the script after applying the fix; otherwise, I encountered a syntax error.

Reply:
Otherwise, I agree with the new approach.

Reply:
Ofc! Double quotes in double quotes 😅 can be wierd

Reply:
But wierd that my Version of Python didnt complain 🤔v3.12

Reply:
I'm using 3.11.5; perhaps there was a relevant change. If it's going to land in the release version later anyway, it doesn't matter. Feel free to revert the fix. I'll simply update my environment to 3.12.

Reply:
added more future proof fix by introducing variable :)